extra_docs/jfs/README.md

```diff
@@ -0,0 +1,209 @@
+# Jexia Fileset Module (JFS)
+
+JFS is used for the uploading, fetching and manipulating files as any other user data. Basically, fileset has the same 
+functionality as dataset, but provides a way to maintain large data objects (files).  
+Before uploading any file you need to manually create a fileset with web management application 
+[app.jexia.com](https://app.jexia.com).
+You can also set AWS bucket and credentials so that your files will be automatically uploaded to the Amazon storage.
+
+### [Initialize](#init) 
+To use fileset features, just add fileOperations module to the client initialization. Notice, that for Node.js
+you have to import fileOperations from *jexia-sdk-js/node*, but if you running your application in browser you need to
+use *jexia-sdk-js/browser* path. It's because file format is different under these two platforms.
+```typescript
+import { jexiaClient, fileOperations } from 'jexia-sdk-js/node'; // 'jexia-sdk-js/browser' for browser applications
```

Comment:
please use double quotes to keep consistency with _most_ of the code (and all other examples)

Reply:
done